_clients/javascript/getting-started.md

```diff
@@ -142,18 +143,22 @@ search().catch(console.log);
 
 ## Circuit Breaker
 
-The `memoryCircuitBreaker` parameter in the [Cluster Settings API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/) gives you the ability to reject large query responses where the size of the response could crash OpenSearch Dashboards. To set the Circuit Breaker setting, use the `POST _cluster/settings` API operation on your active JS OpenSearch cluster.
+Helper function to check if memory circuit breaker enabled and the response payload is too large to fit into available heap memory.
```

Comment:
This doesn't read very well. Maybe make a higher level section called "Client Options" and a sub-section "Circuit Breaker" and then this line can just be removed?